### ISSUE TITLE
Oracle OMOP Phenotype script v3.1 error

### ISSUE THREAD

**Issue by kirenotneb** [National-COVID-Cohort-Collaborative/Phenotype_Data_Acquisition#187]
The v3.1 Oracle OMOP Phenotype script has invalid psql syntax for the `UPDATE` statements to populate the `N3C_CONTROL_MAP`. When running it reports that the "SQL statement is not complete".

For example:
```
@@ -942,7 +942,7 @@ SELECT person_id, 1 as buddy_num, NULL

 -- Match #1 - age, sex, race, ethnicity
 UPDATE @resultsDatabaseSchema.N3C_CONTROL_MAP
-SET control_person_id = y.control_pid
+SET control_person_id = (SELECT y.control_pid
 FROM
 (SELECT cases.person_id as case_pid, cases.buddy_num bud_num, controls.person_id control_pid
        FROM (SELECT subq.*
@@ -994,7 +994,7 @@ FROM
  ) y
 WHERE control_person_id IS NULL
 AND case_person_id = y.case_pid
-AND buddy_num = y.bud_num
+AND buddy_num = y.bud_num )
 ;
```

**Comment by marshallclark** [comment]
Thanks for letting us know.  working on a fix.

**Comment by rtmill** [comment]
Closed by #189
